### PR TITLE
OBSDOCS-460: Clean up about page and old references

### PR DESCRIPTION
**Pull request by abrennan89**
Version(s):
4.11+

Issue:
https://issues.redhat.com/browse/OBSDOCS-460

Link to docs preview:
https://66606--docspreview.netlify.app/openshift-enterprise/latest/logging/cluster-logging

QE review:
- [x] QE has approved this change.

Additional information:
Breaking out some of the changes from https://github.com/openshift/openshift-docs/pull/66171 to reduce the size of that PR 🛩️

### CODE REVIEW

File: modules/logging-architecture-overview.adoc [hunk 1]
```diff
@@ -6,15 +6,25 @@
...
+include::snippets/logging-fluentd-dep-snip.adoc[]
+--
+
+Log store:: The log store stores log data for analysis, and is the default output for the log forwarder. You can use the default LokiStack log store, the legacy Elasticsearch log store, or forward logs to additional external log stores.
++
+--
+include::snippets/logging-elastic-dep-snip.adoc[]
+--
+
+Visualization:: You can use a UI component to view a visual representation of your log data. The UI provides a graphical interface for searching, querying, and viewing stored logs, as well as dashboards created using aggregated log data from your cluster. The recommended UI is provided by enabling the {product-title} console plugin, but you can also use Kibana.
```
Comment (jcantrill) [inline modules/logging-architecture-overview.adoc:23]: Console does not support dashboards and I don't know if that is a future feature, but we should strike that from the verbiage or some how associate it with "kibana"

File: modules/logging-architecture-overview.adoc [hunk 2]
```diff
@@ -24,6 +34,4 @@ The {logging-title} collects container logs and node logs. These are categorized
 
 * `audit` - Logs generated by auditd, the node audit system, which are stored in the */var/log/audit/audit.log* file, and logs from the `auditd`, `kube-apiserver`, `openshift-apiserver` projects, as well as the `ovn` project if enabled.
 
-The logging collector is a daemonset that deploys pods to each {product-title} node. System and infrastructure logs are generated by journald log messages from the operating system, the container runtime, and {product-title}.
-
-Container logs are generated by containers running in pods running on the cluster. Each container generates a separate log stream. The collector collects the logs from these sources and forwards them internally or externally as configured in the `ClusterLogForwarder` custom resource.
+System and infrastructure logs are generated by journald log messages from the operating system, the container runtime, and {product-title}.
```
Comment (jcantrill) [inline modules/logging-architecture-overview.adoc:37]: Infrastructure from a logging perspective are:

* journald message from the node (OS logs)
* container logs from the infra NS (default, kube*,openshift*). 

We may not want to explicitly define the list of NS but we should at least note that some "infrastructure" logs are container workloads that come from "infrastructure" namespaces.

[the remark this answers is not in the document]
Comment (JoaoBraveCoding) [reply modules/logging-architecture-overview.adoc:37]: Also in the `infrastructure` bullet, what do you think of... I changed the beginning part to illustrate better that `infrastructure` logs are comprised of logs from two sources. If you feel it's more confusing just ignore this suggestion.

```
* `infrastructure` - Logs generated by both infrastructure components running on the cluster and by {product-title} nodes, such as journald logs. Infrastructure components are pods that run in the `openshift*`, `kube*`, or `default` projects.
```

**Review by JoaoBraveCoding — COMMENTED** [review]
Overall lgtm, just a comment

File: modules/logging-architecture-overview.adoc [hunk 3]
```diff
@@ -6,15 +6,25 @@
...
-* `Collector` - Reads container log data from each node and forwards log data to configured outputs.
-
-* `Store` - Stores log data for analysis; the default output for the forwarder.
-
-* `Visualization` - Graphical interface for searching, querying, and viewing stored logs.
-
-These components are managed by Operators and Custom Resource (CR) YAML files.
+The major components of the {logging} are:
+
+Collector:: The collector is a daemonset that deploys pods to each {product-title} node. It collects log data from each node, transforms the data, and forwards it to configured outputs. The recommended collector implementation is the Vector collector.
```
Comment (agantony) [inline modules/logging-architecture-overview.adoc:11]:
```suggestion
Collector:: The collector is a daemonset that deploys pods on each {product-title} node. It collects log data from each node, transforms the data, and forwards it to configured outputs. Use the Vector collector as the collector implementation.
```

File: modules/logging-architecture-overview.adoc [hunk 4]
```diff
@@ -6,15 +6,25 @@
...
++
+--
+include::snippets/logging-fluentd-dep-snip.adoc[]
+--
+
+Log store:: The log store stores log data for analysis, and is the default output for the log forwarder. You can use the default LokiStack log store, the legacy Elasticsearch log store, or forward logs to additional external log stores.
```
Comment (agantony) [inline modules/logging-architecture-overview.adoc:17]:
```suggestion
Log store:: The log store stores log data for analysis and is the default output for the log forwarder. You can use the default LokiStack log store, the legacy Elasticsearch log store, or forward logs to additional external log stores.
```

File: modules/logging-architecture-overview.adoc (hunk 1 above)
Comment (agantony) [inline modules/logging-architecture-overview.adoc:23]:
```suggestion
Visualization:: You can use a UI component to view a visual representation of your log data. The UI provides a graphical interface to search, query, and view stored logs, as well as dashboards created from your cluster's aggregated log data. Enable the {product-title} console plugin, but you can also use Kibana.
```

File: modules/logging-architecture-overview.adoc (hunk 2 above)
Comment (agantony) [inline modules/logging-architecture-overview.adoc:37]:
```suggestion
Journald log messages from the operating system, container runtime, and {product-title} generate system and infrastructure logs.
```

Comment (agantony) [reply modules/logging-architecture-overview.adoc:37]: not sure if the technical meaning changes here.

File: snippets/logging-kibana-dep-snip.adoc [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+//
+// Text snippet included in the following modules:
+//
+//
+
+:_content-type: SNIPPET
+
+[NOTE]
+====
+The Kibana web console is now deprecated and will be removed in a future logging release.
```
Comment (agantony) [inline snippets/logging-kibana-dep-snip.adoc:13]:
```suggestion
The Kibana web console is deprecated and is planned to be removed in a future logging release.
```

**Review by agantony — COMMENTED** [review]
Added a few suggestions for your consideration; overall lgtm!

**Comment by anpingli** [comment]
Vector support FIPS from 5.5. https://issues.redhat.com/browse/LOG-2460 Shall I file another ticket?

**Comment by abrennan89** [comment]
@anpingli no that's perfectly fine, I can update it in this PR. Thank you! 🙏🏻

File: modules/logging-architecture-overview.adoc (hunk 2 above)
Comment (abrennan89) [reply modules/logging-architecture-overview.adoc:37]: Decided to make some additional updates to the infra bit based on your and @JoaoBraveCoding's suggestions, thank you 😄 let me know if that's better 🙏🏻

Comment (abrennan89) [reply modules/logging-architecture-overview.adoc]: Updated and removed this section anyway

**Comment by abrennan89** [comment]
@anpingli removed the FIPS note. I also have another PR in progress to clean up the collector / vector docs more generally https://github.com/openshift/openshift-docs/pull/66628

File: modules/logging-architecture-overview.adoc [hunk 6]
```diff
@@ -6,24 +6,30 @@
...
+include::snippets/logging-fluentd-dep-snip.adoc[]
+--
+
+Log store:: The log store stores log data for analysis and is the default output for the log forwarder. You can use the default LokiStack log store, the legacy Elasticsearch log store, or forward logs to additional external log stores.
++
+--
+include::snippets/logging-elastic-dep-snip.adoc[]
+--
+
+Visualization:: You can use a UI component to view a visual representation of your log data. The UI provides a graphical interface to search, query, and view stored logs. The {product-title} web console UI is provided by enabling the {product-title} console plugin, but you can also use Kibana.
```
Comment (anpingli) [inline modules/logging-architecture-overview.adoc:23]: you have to use Kibana  to show data on elasticsearch. You can not use kibana for Lokistack.

When lokistack is the default storage, the {product-title} web console UI is provided by enabling the {product-title} console plugin
When elasticsearch is used as default storage,  You have to use Kibana.

File: modules/logging-architecture-overview.adoc [hunk 7]
```diff
@@ -6,24 +6,30 @@
...
+Application logs:: Container logs generated by user applications running in the cluster, except infrastructure container applications.
 
-The logging collector is a daemonset that deploys pods to each {product-title} node. System and infrastructure logs are generated by journald log messages from the operating system, the container runtime, and {product-title}.
+Infrastructure logs:: Container logs generated by infrastructure namespaces: `openshift*`, `kube*`, or `default`, as well as journald messages from nodes.
 
-Container logs are generated by containers running in pods running on the cluster. Each container generates a separate log stream. The collector collects the logs from these sources and forwards them internally or externally as configured in the `ClusterLogForwarder` custom resource.
+Audit logs:: Logs generated by auditd, the node audit system, which are stored in the */var/log/audit/audit.log* file, and logs from the `auditd`, `kube-apiserver`, `openshift-apiserver` projects, as well as the `ovn` project if enabled.
```
Comment (anpingli) [inline modules/logging-architecture-overview.adoc:35]: and audit logs from the `auditd`, `kube-apiserver`, `openshift-apiserver` services. (It is better to replace projects with services.)

**Pull request merged**